### ISSUE TITLE
Seedlet and Varia: Add support for Image block border radius

### ISSUE THREAD

**Issue by alaczek** [Automattic/themes#3140]
Similar to #3093, it would be awesome to add support for the [Image block border-radius setting](https://github.com/WordPress/gutenberg/pull/27667). We've got a couple of Spearhead-based podcasting layouts to be launched shortly that would immediately benefit from this, but it's a very handy design tool in general and we would love to start taking advantage of it as soon as possible.

**Comment by alaczek** [comment]
Actually, it would be great to have this universally available in Seedlet and Varia, and added to new themes going forward.

**Comment by ianstewart** [comment]
I haven't tested it yet but should we be expanding this to our two big parent themes?

- Varia
- Seedlet

So all their child themes inherit this?

**Comment by ianstewart** [comment]
> Actually, it would be great to have this universally available in Seedlet and Varia, and added to new themes going forward.

… and I just noticed @alaczek thought the same thing. I'll update the title. :)

**Comment by kwight** [comment]
@andres-blanco I'm having a hard time finding it, did you have a PR or diff for this?

**Comment by andres-blanco** [comment]
This was my bad, I self assigned this and then I got caught up by something else. 

I started testing this issue, but adding the theme.json in the theme root dir broke the visualization of the block editor. 

I will unassign myself because I think this sprint there are more relevant things to take care of.

**Comment by MaggieCabrera** [comment]
> I started testing this issue, but adding the theme.json in the theme root dir broke the visualization of the block editor.

Yes, we've been there too :D Adding theme.json enables the new layout container and changes how alignments work for the theme and that's a whole other can of worms so we ended up removing theme.json files from all our classic themes for the moment.
